src/Language/PureScript/Sugar/ObjectWildcards.hs

```diff
@@ -45,9 +45,9 @@ desugarObjectConstructors (Module ss coms mn ds exts) = Module ss coms mn <$> ma
     obj <- freshIdent'
     Abs (Left obj) <$> wrapLambda (ObjectUpdate (argToExpr obj)) ps
   desugarExpr (ObjectUpdate obj ps) = wrapLambda (ObjectUpdate obj) ps
-  desugarExpr (Accessor prop u) | isAnonymousArgument u = do
+  desugarExpr (Accessor prop u) | isAnonymousAccessorChain u = do
```

Comment:
@rvion I'm going through PRs for the 0.10 release at the moment. Do you think you'll be able to finish this off for 0.10 or would you prefer to leave it until 0.10.1?